provider/src/main/kotlin/org/gradle/kotlin/dsl/accessors/CodeGenerator.kt

```diff
@@ -22,142 +22,142 @@ import org.jetbrains.kotlin.lexer.KtTokens
 
 internal
 fun ProjectSchema<TypeAccessibility>.forEachAccessor(action: (String) -> Unit) {
-    extensions.forEach { (name, type) ->
-        extensionAccessorFor(name, type)?.let(action)
+    extensions.map(::typedAccessorSpecFor).forEach { spec ->
+        extensionAccessorFor(spec)?.let(action)
     }
-    conventions.forEach { (name, type) ->
-        if (name !in extensions) {
-            conventionAccessorFor(name, type)?.let(action)
+    conventions.map(::typedAccessorSpecFor).forEach { spec ->
+        if (spec.name.original !in extensions) {
+            conventionAccessorFor(spec)?.let(action)
         }
     }
-    configurations.forEach { name ->
-        configurationAccessorFor(name)?.let(action)
+    configurations.map(::accessorNameSpecFor).forEach { spec ->
```

Comment:
:x: Same as above with regards the `For` suffix. `configurations.map(::accessorNameSpec).forEach { ...` reads a little better.